sklearnex/dispatcher.py

```diff
@@ -143,6 +146,10 @@ def get_patch_map_core(preview=False):
         )
         from .linear_model import LinearRegression as LinearRegression_sklearnex
         from .linear_model import LogisticRegression as LogisticRegression_sklearnex
+        from .manifold import TSNE as TSNE_sklearnex
+        from .metrics import pairwise_distances as pairwise_distances_sklearnex
```

Comment:
I guess elasticnet and ridge would be needed here as well? I am a bit confused on if this is necessary to add to patching but if so then probably need all? @Alexsandruss any idea why these would not be in here?

Reply:
In my understanding, if we wrap this functionality for functional support with dpnp/dpctl, then it matters to add it to the patching. Since now the possibility of wrapping appeared only in sklearnex level, it makes sense. If there are any objections or arguments against, please share

Reply:
All decorated sklearnex estimators should be added in patching map, otherwise unchanged daal4py estimators will be used in it.

Reply:
So this confirms that it is need to update the patching map.

Reply:
Yes so wouldn't it make sense to also add ridge and elasticnet to the patching if we are adding tsne?

Reply:
Hey, sorry I misunderstood your original comment. Very good catch. I thought I have already added them. 
Actually we **should** add them to the map. I will update map patching. 

I would like also understand why this tests passed without it. It is weird little bit for me that dpnp/dpctl tests passed… I will check them all once again and report. Potentially there could be an issue.

Reply:
So there was not any `dpnp`/`dpctl` testing with GPU for `Ridge`, `ElasticNet` and `Lasso`. That is why old patching worked fine without `usm_ndarray_support` decoration. Will cover this asap.

Reply:
Pleas see: https://github.com/intel/scikit-learn-intelex/pull/1891